tests/unit/test_toolchain.cpp

```diff
@@ -92,9 +104,11 @@ void TestToolchain(void)
     test_packed_enum();
 }
 
+#ifndef _WIN32
```

Comment:
If there is a common header for all of these tests, then one solution would be something like this:

``` c
#ifdef _WIN32
#define main disabled_main
int disabled_main(void);
#endif
```